frontend/packages/dev-console/src/components/import/builder/BuilderImageSelector.tsx

```diff
@@ -37,11 +37,11 @@ const BuilderImageSelector: React.FC<BuilderImageSelectorProps> = ({
   );
 
   React.useEffect(() => {
-    if (builderImageCount === 1) {
+    if (!selected.value && builderImageCount === 1) {
       const image = _.find(builderImages);
       handleImageChange(image.name);
     }
-  }, [builderImageCount, builderImages, handleImageChange]);
+  }, [builderImageCount, builderImages, handleImageChange, selected.value]);
 
   if (builderImageCount === 1) {
```

Comment:
Can this be combined with line 40?

Reply:
No, these are two different cases. The first one is useEffect hook that is being used in case of s2i where there is only 1 builderImage and the second one is being used for other cases where there is a need for more than 1 builderImage like in git import.